initialization/timing.py

```diff
@@ -9,27 +10,28 @@ def init_timing(app):
     app.before_request(start_request_timing)
     app.after_request(log_request_timing)
 
-    sql_event.listens_for(Engine, "before_cursor_execute", start_db_timing)
-    sql_event.listens_for(Engine, "after_cursor_execute", log_db_timing)
+    sql_event.listens_for(Engine, "before_cursor_execute")(start_db_timing)
+    sql_event.listens_for(Engine, "after_cursor_execute")(log_db_timing)
```

Comment:
This was a bug, it's why database timing wasn't working. Oddly, I think that it used to work, so maybe it depends on the version of SQLAlchemy.